Fauna/Exceptions/ConstraintFailure.cs

```diff
@@ -3,21 +3,39 @@
 
 namespace Fauna.Exceptions;
 
+/// <summary>
+/// A class representing a  <see href="https://docs.fauna.com/fauna/current/reference/fsl/check/">constraint failure</see> from Fauna.
+/// </summary>
```

Comment:
Got a doxygen warning for this. This clears it up:
```suggestion
/// <summary>
/// A class representing a <a href="https://docs.fauna.com/fauna/current/reference/fsl/check/">constraint failure</a> from Fauna.
/// </summary>
```